imperial_coldfront_plugin/views.py

```diff
@@ -531,17 +532,33 @@ def add_rdf_storage_allocation(request):
                 quantity=1,
                 start_date=timezone.now().date(),
                 end_date=form.cleaned_data["end_date"],
+                is_changeable=True,
             )
             rdf_allocation.resources.add(rdf_resource)
 
             storage_quota_attribute_type = AllocationAttributeType.objects.get(
                 name="Storage Quota (GB)"
             )
-            AllocationAttribute.objects.create(
+            quota_attribute = AllocationAttribute.objects.create(
                 allocation_attribute_type=storage_quota_attribute_type,
                 allocation=rdf_allocation,
                 value=storage_size_gb,
             )
+            AllocationAttributeUsage.objects.create(
+                allocation_attribute=quota_attribute, value=0
+            )
+
+            files_quota_attribute_type = AllocationAttributeType.objects.get(
+                name="Files Quota"
+            )
+            files_attribute = AllocationAttribute.objects.create(
+                allocation_attribute_type=files_quota_attribute_type,
+                allocation=rdf_allocation,
+                value=settings.GPFS_FILES_QUOTA,
```

Comment:
Was this `GPFS_FILES_QUOTA` already defined in settings? For what was it used?

Reply:
It's used [here](https://github.com/ImperialCollegeLondon/imperial_coldfront_plugin/blob/a92b77fad34f9168be9461876d26ada1cfa3a1cd/imperial_coldfront_plugin/views.py#L599-L611) when creating a fileset and setting the initial quota values. It provides the maximum number of files that can be created in a fileset. It's expected to be a fixed value for all filesets.